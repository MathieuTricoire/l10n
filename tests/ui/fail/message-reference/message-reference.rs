l10n::init!();

fn main() {
    let _ = l10n::message!("resource", "baz");
}
