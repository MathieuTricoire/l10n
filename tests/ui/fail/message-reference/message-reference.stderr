error: missing arguments: "bar-var", "baz-var", "foo-var" for resource: "resource" and key: "baz"
 --> tests/ui/fail/message-reference/message-reference.rs:4:40
  |
4 |     let _ = l10n::message!("resource", "baz");
  |                                        ^^^^^
